### DISCUSSION TITLE
What do you want out of Freedom Stack?

### DISCUSSION THREAD

**Discussion by cameronapak** [cameronapak/freedom-stack#34] [Q&A]
Hey! I'm Cam Pak, creator and maintainer of Freedom Stack, a full-stack Astro starter kit that feels freeing and is free.

Freedom Stack is getting better day by day, and I have one question for you:

### What do you want out of Freedom Stack?

Write your answers, and I'd be happy to dream and discuss!

**Comment by onurcanbaysal** [comment]
Hi Cameron, I am lucky that I saw your comment on Reddit. Honestly, your websites are amazing! Do you open source faith.tools template ? For a long time, I am searching good listing/directory template. 

Currently I am using https://astro.build/themes/details/minted-directory/ , thanks to Mark Bruderer.

**Reply by cameronapak** [reply]
Hey, thank you! I don't have faith.tools open-sourced but might should one day.

I know you're wanting an Astro directory, but I recently stumbled upon https://www.minteddirectory.com/ _(oddly enough the same name as your link)_ but it's written in Nuxt (i.e. Vue). Seems beautiful and loaded... and free!
